### ISSUE TITLE
Bring back the Long Context chat

### ISSUE THREAD

**Issue by i0x404** [cursor/cursor#2447]
Hello, 
Please bring back the Long Context Chat that was removed in v0.43.x, as it was better for responding to a longer codebase.

**Comment by StylezDevops** [comment]
agreed it was WAY better. this thing is DUMB now. it is like talking to a ten year old. tasks that claude sonnet previously handled with ease (particularly when dealing with a codebase/long context chat) - now it just asks loads of dumb follow up questions - "would you like me to proceed?" - then does half the task and asks more dumb questions with "would you like to continue" - it's awful and to the point i might even just cancel the subscription. what's the point of it now? I can get this level of stupidness for free.
